### PR TITLE
PIMOB-1281: Fix chevron color

### PR DESCRIPTION
**Pull request by ehab-al-cko**
[Jira ticket ](https://checkout.atlassian.net/browse/PIMOB-1281?atlOrigin=eyJpIjoiZWI4MGZhYjllNGFjNDhmY2JkZDM2YTc0MTEwOWQ2YTgiLCJwIjoiaiJ9)
## Proposed changes

Fix Chevron image color 

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ class ImageContainerView: UIView {
     }
 
     func update(with image: UIImage?, tintColor: UIColor? = nil) {
-        imageView?.image = image
+        imageView?.image = image?.withRenderingMode(.alwaysTemplate)
```
Comment (deepesh-vasthimal-cko) [inline Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift:23]: Question : Did you not need a new image form Alex our designer ? or you are just changing the tint color

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift (hunk 1 above)
Comment (ehab-al-cko) [reply Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift:23]: just changing the tint color

**Review by alex-ioja-yang-cko — DISMISSED** [review]
There just seem to be some unintended indentation changes.

File: Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift (hunk 1 above)
Comment (harry-brown-cko) [reply Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift:23]: can we make this only with `.withRenderingMode(.alwaysTemplate)` if the `tintColor` is not nil? This current change will break card scheme icons once they are PDFs

**Review by harry-brown-cko — CHANGES_REQUESTED** [review]
https://github.com/checkout/frames-ios/pull/193/files#r914960558

File: Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift (hunk 1 above)
Comment (ehab-al-cko) [reply Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift:23]:
```suggestion
        imageView?.image = tintColor == nil ? image ? image?.withRenderingMode(.alwaysTemplate)
```

**Review by harry-brown-cko — APPROVED** [review]

**Comment by alex-ioja-yang-cko** [comment]
Can we please not abuse the merge system? [PR-194](https://github.com/checkout/frames-ios/pull/194) was only with 1 approval, not 3, but by having it merged here and the approvals not being reset on commit, it would seem as if all changes here have been approved by 3 people, which is not accurate.

This is not the first time, so please try to merge in order.

If D -> C -> B -> A -> main, then you would have to merge A, then B, then C and finally D. That would be closer to the order reviews and PRs were opened and built!

**Pull request merged**